packages/devtools_app/lib/src/screens/inspector/inspector_service.dart

```diff
@@ -388,25 +388,26 @@ class InspectorService extends InspectorServiceBase {
   }
 
   Future<void> _updateLocalClasses() async {
-    localClasses.clear();
-    if (_rootDirectories.value.isNotEmpty) {
-      final isolate = inspectorLibrary.isolate!;
-      for (var libraryRef in isolate.libraries!) {
-        if (isLocalUri(libraryRef.uri!)) {
-          try {
-            final Library library = await inspectorLibrary.service
-                .getObject(isolate.id!, libraryRef.id!) as Library;
-            for (var classRef in library.classes!) {
-              localClasses[classRef.name!] = classRef;
-            }
-          } catch (e) {
-            // Workaround until https://github.com/flutter/devtools/issues/3110
-            // is fixed.
-            assert(serviceManager.connectedApp!.isDartWebAppNow!);
-          }
-        }
-      }
-    }
+    // TODO(https://github.com/flutter/devtools/issues/4393)
```

Comment:
can you reference this bug from the isLocalClass getter too? Just some someone in the future doesn't try to use isLocalClass and get confused why it's always false.

Reply:
great idea!

Reply:
comment added :)